$OpenBSD: patch-include_vlc_playlist_h,v 1.1 2017/05/22 10:22:25 espie Exp $
Keep them outside of extern C, vlc_atomic.h will use std::atomic.

Index: include/vlc_playlist.h
--- include/vlc_playlist.h.orig
+++ include/vlc_playlist.h
@@ -24,12 +24,12 @@
 #ifndef VLC_PLAYLIST_H_
 #define VLC_PLAYLIST_H_
 
+#include <vlc_input.h>
+#include <vlc_events.h>
+
 # ifdef __cplusplus
 extern "C" {
 # endif
-
-#include <vlc_input.h>
-#include <vlc_events.h>
 
 TYPEDEF_ARRAY(playlist_item_t*, playlist_item_array_t)
 
